networking_dvs/plugins/ml2/drivers/mech_dvs/agent/dvs_agent.py

```diff
@@ -178,6 +183,14 @@ def book_port(self, port, network_segments, network_current):
 
         return {"bridge_name": pg["name"]}
 
+    def mtu_update(self, network_mtu, dvs_mtu, dvs, network_current):
```

Comment:
This is a switch-wide setting. A VM that expects, say 1500, will start to misbehave if hit with 9000-sized frames for whatever reason.

Reply:
The difference is now, that the VM will drop the jumbo-frame instead the switch. Which is perfectly fine either way, as the jumbo-frame is "illegal" anyway.

Reply:
That's true, but then you can set 9000 to all distributed switches upfront rather than handling it in a lazy/dynamic manner.

Reply:
From my point-of-view, that is something which should happen, but it should happen outside of the driver.

The problem with setting it up-front _in_ the driver is, to what value do I set it?

I don't think, a hard-coded value would be sensible, so it would need another configuration value, which might be add odds with whatever somewhere else is configured. So it would introduce another source of error.

By setting it lazily, I am ensuring that the dvs is configured consistent with the rest of the network configuration.